src/dbnode/integration/setup.go

```diff
@@ -883,6 +889,76 @@ func (ts *testSetup) SchemaRegistry() namespace.SchemaRegistry {
 	return ts.schemaReg
 }
 
+// InitializeBootstrappersOptions supplies options for bootstrapper initialization.
+type InitializeBootstrappersOptions struct {
+	CommitLogOptions commitlog.Options
+	WithCommitLog    bool
+	WithFileSystem   bool
+}
+
+func (o InitializeBootstrappersOptions) validate() error {
+	if o.WithCommitLog && o.CommitLogOptions == nil {
+		return errors.New("commit log options required when initializing a commit log bootstrapper")
+	}
+	return nil
+}
+
+func (ts *testSetup) InitializeBootstrappers(opts InitializeBootstrappersOptions) error {
```

Comment:
Refactored bootstrapper initializing to be _mostly_ done in a centralized place. This would be useful in enterprise tests as well.

Reply:
+1